admin/sql/create_tables.sql

```diff
@@ -104,17 +104,20 @@ CREATE TABLE dataset_snapshot (
 );
 
 CREATE TABLE dataset_eval_jobs (
-  id                UUID,
-  snapshot_id       UUID                     NOT NULL, -- FK to snapshot
-  status            eval_job_status          NOT NULL DEFAULT 'pending',
-  status_msg        VARCHAR,
-  options           JSONB,
-  training_snapshot INT,                     -- FK to dataset_eval_sets
-  testing_snapshot  INT,                     -- FK to dataset_eval_sets
-  created           TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
-  updated           TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
-  result            JSONB,
-  eval_location     eval_location_type       NOT NULL DEFAULT 'local'
+  id                   UUID,
+  snapshot_id          UUID                     NOT NULL, -- FK to snapshot
+  status               eval_job_status          NOT NULL DEFAULT 'pending',
+  status_msg           VARCHAR,
+  options              JSONB,
+  training_snapshot    INT,                     -- FK to dataset_eval_sets
+  testing_snapshot     INT,                     -- FK to dataset_eval_sets
+  created              TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  updated              TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  result               JSONB,
+  eval_location        eval_location_type       NOT NULL DEFAULT 'local',
+  c_value              VARCHAR,
```

Comment:
Should we consider having default values for these new columns?

Reply:
we already have the `options` field which stores some user-provided settings for an evaluation. I think that we should store these items in there instead of having new parameters.

Reply:
As we also have a parameter of `evaluation_location` here: https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/forms.py#L31, which is enabled to display on the acousticbrainz site when https://github.com/metabrainz/acousticbrainz-server/blob/master/default_config.py#L62 is turned `True`.
Similarly, the c, gamma and preprocessing parameters are added separately.
Should I change it? :)

Reply:
Storing in the `options` field would remove the need for a schema change. I'd prefer it but let us wait for @alastair to confirm.

Reply:
@rsh7 I think we should go with moving the values in the new columns to options.